open-company/open-company-web
# Verify route fix

Card: https://trello.com/c/r5FPce9o

Bug: when visiting /verify route without a token we show a blank screen and an error is reported to sentry.

Fix: redirect to user org or homepage (in case user is not logged in) when visiting /verify without a token.

To test:
- signup
- wait for the verification email
- click the link
- [x] make sure you have verified email (open QSG for example)
- now visit /verify (without a token as get parameter)
- [ ] did it redirect you back to your org? Good
- logout
- visit /verify
- [x] did it redirect you to homepage? Good
